IDE/GCC-ARM/Makefile.common

```diff
@@ -113,18 +114,28 @@ SRC_C += ../../wolfcrypt/src/wolfcrypt_last.c
 endif
 
 # wolfCrypt Additional
+SRC_C += ../../wolfcrypt/src/asm.c
+SRC_C += ../../wolfcrypt/src/async.c
 SRC_C += ../../wolfcrypt/src/asn.c
+SRC_C += ../../wolfcrypt/src/blake2s.c
```

Comment:
Why not blade2b.c also?

Reply:
That was already included but in a lower section. I didn't want to re-mesh those in this list.